src/main/java/com/greenfoxacademy/goddesstribesbackend/models/dtos/TokenMessage.java

```diff
@@ -3,11 +3,11 @@
 public class TokenMessage {
 
   private String status;
-  private String token;
+  private String tribes_token;
```

Comment:
As agreed today, please be consistent with camelCase!
(if it's bad in swagger, adjust that instead of this.)